views/widget.html

```diff
@@ -1,6 +1,8 @@
 {% set widget = data.widget %}
 {% if widget.linkType == 'page' %}
   {% set path = widget._linkPage._url %}
+{% elif widget.linkType == 'file' %}
+  {% set path = widget._linkFile._url%}
```

Comment:
Does `._url` work on an `apostrophe-file` join? Or do you need to use the `apos.attachments()` template helper to pull it out?

Reply:
@grdunn It does work, recent feature addition.

Reply:
👍